### PR TITLE
WIP: Mothur toolshed dependencies

### PR DESCRIPTION
**Pull request by shiltemann**
adds toolshed dependencies definition for mothur as well as legacy blast version 2.2.22 package (to match the version used in conda)

### CODE REVIEW

File: tools/mothur/macros.xml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 <macros>
     <xml name="requirements">
         <requirements>
-            <requirement type="package" version="1.36.1">mothur</requirement>
+            <requirement type="package" version="1.36.1" owner="iuc">mothur</requirement>
```
Comment (bgruening) [inline tools/mothur/macros.xml]: You don't need an owner here.

Comment (shiltemann) [inline tools/mothur/macros.xml]: hmm ok, I took a guess, when I tested this in testtoolshed it tried to reinstall the mothur dependency for the second of the separate mothur tools I tried to install ..maybe I've got something else misconfigured?

Comment (shiltemann) [inline tools/mothur/macros.xml]: or do I need to make it a package if we're going to have multiple separate tools?

Comment (bgruening) [inline tools/mothur/macros.xml]: This would be great otherwise we will install mothur 125 times :)

Comment (shiltemann) [inline tools/mothur/macros.xml]: lol, yeah that would be bad, okay, back to the drawing board :)

**Comment by shiltemann** [comment]
@bgruening ok, I created a separate package for mothur, tested this in testtoolshed and it seems to work, I can't tell what travis is failing on though?

**Comment by bgruening** [comment]
LGTM. Travis is failing mainly because of the missing/wrong datatypes that are missing in the 16.4 release I guess.

**Pull request merged**